rest/server_context.go

```diff
@@ -144,6 +150,8 @@ func (sc *ServerContext) Close() {
 	}
 
 	sc.stopStatsReporter()
+	sc.stopStatsLogger()
```

Comment:
Should we be deprecating/removing StatsReporter?

Reply:
I can remove as part of this PR.  Was this exposed in docs or config that requires deprecation?  I'm not too familiar w/ this feature.

Reply:
It predates me as well, but I've never heard of anyone using it (or us documenting it) - I think it's reasonable for the new functionality to be treated as a functional replacement for this and just take it right out.

Reply:
Removed